### ISSUE TITLE
Long Tests 

### ISSUE THREAD

**Issue by Chris-Yarland-Price** [a85/POSTMan-Chrome-Extension#4181]
## App Details:

```
Postman for Linux
Version 5.5.2
linux 4.13.0-32-generic / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: 
Encountered recently but i've only used postman for about 5 weeks
 2. Expected behaviour: Long test names should not cause the runner to collapse. So i am trying to get the response body included in the runner results for easy viewing however by doing the following it causes the viewer to completely fall apart. 

pm.test("Response Body is " + responseBody, function(){
    true;
});

As you can see in the screenshot this causes the viewer to hide behind other values on the field. 

 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
 4. Screenshots (if applicable)
![error](https://user-images.githubusercontent.com/35267782/36159709-49eaca3e-10d7-11e8-98df-ccaabf6bfe39.png)

pm.test("Response Body is " + responseBody, function(){
    true
})

**Comment by kamalaknn** [comment]
@Chris-Yarland-Price 

Thanks for reporting this. We will look into this.

**Comment by Chris-Yarland-Price** [comment]
This also happens for newman using HTML almost the same exact problem

**Comment by sdnts** [comment]
@Chris-Yarland-Price, this should be fixed now in the latest version (6.0.10). Test names should be truncated correctly now.
I'm closing this. Please let me know if you still see this, and I'll reopen.

**Comment by Chris-Yarland-Price** [comment]
Hi This has fixed the bug but doesn't really help if you can't actually read what you are trying to do
